ProjectFiles/Assets/Scripts/Network/ServerConfig.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Network
```

Comment:
Not sure if this should be in the network namespace? At the moment its only storing things about the network but we could end up expanding it further in the future

Reply:
Maybe just something to bear in mind for the future